pkg/hoarder/discoverer.go

```diff
@@ -181,15 +184,22 @@ func (discoverer *CidDiscoverer) addProviderRecordsHttp(addProviderWG *sync.Wait
 
 			cidInfo := models.NewCidInfo(cidIn, discoverer.ReqInterval, discoverer.StudyDuration, config.JsonFileSource,
 				discoverer.CidSource.Type(), "")
+
+			cidInfo.AddPublicationTime(tr.PublicationTime)
+			cidInfo.AddProvideTime(tr.ProvideTime)
+			cidInfo.AddCreator(tr.Creator)
```

Comment:
`NewCidInfo` has an argument to specify the Creator of the CID
```go
func NewCidInfo(
	id cid.Cid,
	reqInt time.Duration,
	studyDurt time.Duration,
	contentType, source string,
	creator peer.ID) *CidInfo
``` 
do we still need an `AddCreator()` method in for it?

Reply:
It is redundant, but I don't see any harm in it.

Reply:
Normally, we create the `CidInfo` when the CID has been already created, so by the time we create the CidInfo, we already know who was the creator.
I'm just afraid of rewriting the creator of the CID somewhere else, so removing the method would make the interface easier to follow (imo)